app/models/order.rb

```diff
@@ -1,13 +1,19 @@
 class Order < ApplicationRecord
-  after_initialize :set_status
+  after_initialize :set_defaults
 
   validates :status, presence: true
   has_many :order_items
   has_many :items, through: :order_items
   belongs_to :user
   enum status: %w(Pending Shipped Delivered)
 
-  def set_status
+  def set_defaults
```

Comment:
Orders are setting defaults of delivery_date. This is incomplete and ideally will be at least 5 days from the day the order was placed